tests/test_page_turn_loop.py

```
@@ -108,60 +222,72 @@ class TestLoopErrorHandling:
     
     def test_stability_timeout_handling(self, mocker):
         """安定判定タイムアウト時の処理確認"""
-        mock_stability = mocker.patch('src.core.stability.StabilityChecker')
-        mock_capture = mocker.patch('src.core.capture.CaptureManager')
+        mock_stability = mocker.patch('src.core.capture_loop.StabilityChecker')
+        mock_capture = mocker.patch('src.core.capture_loop.CaptureManager')
```

Ruff (0.12.2)

Local variable `mock_capture` is assigned to but never used

Remove assignment to unused variable `mock_capture`

(F841)